build.sbt

```diff
@@ -34,11 +34,9 @@ val noPublish = List(
 )
 
 // === CI/CD settings ===
-val scala213 = "2.13.11"
-val scala3   = "3.3.0"
+val scala3   = "3.4.2"
 
-ThisBuild / scalaVersion       := scala213
-ThisBuild / crossScalaVersions := List(scala213, scala3)
+ThisBuild / scalaVersion       := scala3
```

Comment:
What's the motivation for dropping Scala 2?

As a FLOSS project, limiting the potential user base is a decision we shouldn't take without proper consideration. If it's only the overhead of applying some smaller syntax fixes in both versions, I'm rather leaning towards keeping Scala 2.

Reply:
Let's call it a proposal. 

It just reduces the complexity a little, true. But as Scala 3 has been around for quite some time by now, it's hard to believe that potential users are out there who still rely on Scala 2.
How many users do we currently have anyway?

One benefit is a slightly simplified build and a bunch less source files. Another one is the ability to use modern Scala 3 syntax all over the place - in documentation and in the source code, which provides easier access to developers who are used to to Scala 3 syntax.

Reply:
According to https://scalasurvey2023.virtuslab.com, Scala 2.13 is still the main version in use. To be honest, I don't know how many are using `fs2-gtfs`, but still I believe it's too early to cut off Scala 2 entirely, given that we only gain "convenience".

As for the docs, we can to a large degree still use Scala 3 syntax. Most Scala 2 developers will be able to translate without problems.

Reply:
Your referred survey also says that 50% already use Scala 3. It doesn't say anything about new projects being started on Scala 2. My thesis: Most devs would nowadays not start anything new on Scala 2.
And if people didn't use this library in their legacy projects until now, it's highly unlikely they'll do so in the future.

A rough idea on how many people are interested in this lib can be obtained here:
https://oss.sonatype.org/#central-stat

I am not an expert on how to interpret the download figures, but I would call them pretty low.

I would actually go as far as to say that we have no active users currently. I don't really see the point of keeping the extra complexity for no measurable benefit. What you called "convenience" translates into less time and therefore less money spent on development.

Reply:
The numbers are indeed not high. People who are using 2.13 can still use this, just not get updates. 

In theory i would like to have a "perfect" open source project which caters to everyone's needs. In practice i feel having it developed further and being used/improved is worth more than scala2.13 support. 

Since @gbastkowski currently has a need for this project and is willing to develop it further, lets not make it harder for the one user we have 🙈

Reply:
I wish there was not the dichotomy between "develop it further" and "keep Scala 2", but rather a "how can we achieve both".

But I accept that my defence of a very small, potentially even empty user group isn't that well-justified, so go ahead.

Reply:
Thanks. As soon as some user asks for Scala 2 support, I'll be the first advocate to add it back again.